python-sdk/nuscenes/eval/panoptic/panoptic_track_evaluator.py

```diff
@@ -129,6 +133,48 @@ def get_panoptic_track_stats(self,
             unique_combo_, counts_combo_ = np.unique(offset_combo_, return_counts=True)
             self.update_dict_stat(cl_intersects, unique_combo_, counts_combo_)
 
+            # Computation for PAT score
```

Comment:
could you help to correct the return typing to `Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray, Dict[int, int], Dict[int, int], np.ndarray]`  in lines 83-84. tks.